pyk/src/pyk/kore/rpc.py

```diff
@@ -265,9 +265,6 @@ def __init__(
         bug_report_id: str | None = None,
         transport: TransportType = TransportType.SINGLE_SOCKET,
     ):
-        if (bug_report is None) != (bug_report_id is None):
-            raise ValueError('bug_report and bug_report_id must be passed together.')
-
```

Comment:
Do you know why this was here before?

Reply:
I'm actually not sure, it was very first introduced here: https://github.com/runtimeverification/k/pull/4480/files

But as far as I can tell, this shouldn't really be an issue. This check, I guess, would make sure that each bug report has a unique user-specified ID. But if they don't have that ID, then the user can supply it separately.